include/inexor/vulkan-renderer/availability_checks.hpp

```diff
@@ -40,34 +32,30 @@ class AvailabilityChecksManager {
     ~AvailabilityChecksManager() = default;
 
     /// @brief Checks if a certain Vulkan instance layer is available on the system.
-    /// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkEnumerateInstanceLayerProperties.html
-    /// @param instance_layer_name The name of the Vulkan instance layer.
-    /// @return true if the Vulkan instance layer is available, false otherwise.
+    /// @param instance_layer_name [in] The name of the Vulkan instance layer.
+    /// @return True if the Vulkan instance layer is available, false otherwise.
```

Comment:
```suggestion
    /// @return `true` if the Vulkan instance layer is available, `false` otherwise.
```
Looks a bit nicer imo.

Reply:
Done for all `@return` commands.